psc-docs/Main.hs

```diff
@@ -31,21 +31,33 @@ import qualified Paths_purescript as Paths
 import System.Exit (exitSuccess, exitFailure)
 import System.IO (hPutStrLn, stderr)
 
+data Format = Markdown
```

Comment:
Probably worth a line comment for those of us (like me) who don't know what these formats are for :)